packages/fxa-payments-server/Gruntfile.js

```diff
@@ -25,13 +26,13 @@ module.exports = function (grunt) {
       // FTL updates on our side that haven't landed yet on the l10n side. We want to test
       // against _our_ latest, and not necessarily the l10n repo's latest.
       'ftl-test': {
-        src: ['src/branding.ftl', 'src/**/*.ftl'],
+        src: ['src/branding.ftl', 'src/**/*.ftl', '../fxa-react/**/*.ftl'],
```

Comment:
This change appears to be pulling strings for testing unintentionally from `packages/fxa-react/lib/test-utils/test.ftl`.

Reply:
Thanks for catching that! I'll update to only pull from fxa-react/src/components